src/nix/lib.nix

```diff
@@ -3,11 +3,13 @@ let
 
   link = url: text: ''[${text}](${url})'';
 
+  composeSpecRev = "55b450aee50799a2f33cc99e1d714518babe305e";
```

Comment:
As of 2023-07-21, this is the latest commit to the `master` branch. `compose-spec` doesn't have any releases or tags yet, so a commit seems most appropriate to ensure documentation links remain valid.